<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Network Rebus.net</author>
 <class>ConfigForm</class>
 <widget class="QDialog" name="ConfigForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting locate database</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="ButtonGroupSearchIn">
     <property name="title">
      <string>DB to search in</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="updatedbConfLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="CheckBoxUseDefaultDB">
          <property name="toolTip">
           <string>If checked, qlocate query the default index DB
define in /etc/updatedb.conf</string>
          </property>
          <property name="whatsThis">
           <string>If checked, qlocate query the default index DB
define in /etc/updatedb.conf</string>
          </property>
          <property name="text">
           <string>Use default DB define in updatedb.conf</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="updatedbConfPathLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="LineEditSearchIn">
            <property name="toolTip">
             <string>If checkbox &quot;Use default DB&quot; is not checked,
this define the index DB to query.</string>
            </property>
            <property name="whatsThis">
             <string>If checkbox &quot;Use default DB&quot; is not checked,
this define the index DB to query.</string>
            </property>
            <property name="maxLength">
             <number>4096</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PushButtonSerachIn">
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBoxUpdatedb">
     <property name="title">
      <string>Create index DB</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="dbOptionsLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabelDBLocate">
          <property name="text">
           <string>Where the database is create :</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dbPathLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="LineEditDBLocate">
            <property name="toolTip">
             <string>Define where the file index DB is create and located.</string>
            </property>
            <property name="whatsThis">
             <string>Define where the file index DB is create and located.</string>
            </property>
            <property name="maxLength">
             <number>4096</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PushButtonDBLocate">
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="indexOptionsLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="TextLabelStartIndexFrom">
          <property name="text">
           <string>Start index from :</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="indexButtonsLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="LineEditStartIndexFrom">
            <property name="toolTip">
             <string>Define the base directory to index (recursive).</string>
            </property>
            <property name="whatsThis">
             <string>Define the base directory to index (recursive).</string>
            </property>
            <property name="maxLength">
             <number>4096</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PushButtonStartIndexFrom">
            <property name="text">
             <string>Change</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="TextLabelDirExclude">
          <property name="text">
           <string>Directory to exclude :</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QListWidget" name="ListBoxDirExclude">
            <property name="toolTip">
             <string>List all directories (recursive) to exclude in index DB.</string>
            </property>
            <property name="whatsThis">
             <string>List all directories (recursive) to exclude in index DB.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="directoryButtonsLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="PushButtonAddDirExclude">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PushButtonDeleteDirExclude">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="TextLabelFsExclude">
          <property name="text">
           <string>Filesystems do we exclude :</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QListWidget" name="ListBoxFsExclude">
            <property name="toolTip">
             <string>List all filesystem to exclude in index DB.</string>
            </property>
            <property name="whatsThis">
             <string>List all filesystem to exclude in index DB.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="filesystemButtonsLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="PushButtonAddFsExclude">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="PushButtonDeleteFsExclude">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="CheckBoxSecurityLevel">
            <property name="toolTip">
             <string>If checked, active the security when the index DB is build.</string>
            </property>
            <property name="whatsThis">
             <string>If checked, active the security when the index DB is build.</string>
            </property>
            <property name="text">
             <string>Security level ( On / Off )</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CheckBoxVerbose">
            <property name="toolTip">
             <string>If checked, files name is print on standard output canal when DB is build.
WARNING this feature significant increase the time for building the index DB.</string>
            </property>
            <property name="whatsThis">
             <string>If checked, files name is print on standard output canal when DB is build.
WARNING this feature significant increase the time for building the index DB.</string>
            </property>
            <property name="text">
             <string>Verbose ( Yes / No )</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="PushButtonCreateDB">
            <property name="toolTip">
             <string>Run the command for build index DB.</string>
            </property>
            <property name="whatsThis">
             <string>Run the command for build index DB.</string>
            </property>
            <property name="text">
             <string>Create DB</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="SpacerCancelQuit">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonConfSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>PushButtonConfSave</tabstop>
  <tabstop>PushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
